Config/CLIParameterConfig.cs

```diff
@@ -86,5 +83,26 @@ public override PackageMeta GetPackageMeta()
                 VersionNumber = options.VersionNumber
             };
         }
+
+        public override PublishConfig GetPublishConfig()
+        {
+            return new PublishConfig()
+            {
+                Repository = options.Repository
+            };
+        }
+
+        public override AuthConfig GetAuthConfig()
+        {
+            if (options.UseSessionAuth)
+            {
+                Console.WriteLine(Yellow("The usage of session auth WILL be deprecated in the future, possibly without warning!"));

```

Comment:
Strictly speaking, it's already deprecated to begin with, it will be _removed_ in a future release without warning.